hunting/okta/docs/queries/initial_access_impossible_travel_sign_on.toml

```diff
@@ -0,0 +1,34 @@
+[hunt]
+author = "Elastic"
+description = """
+This hunting query identifies when a user successfully signs on from more than one country in a 15 minute interval. Adversaries may compromise authentication credentials for users or clients and attempt to authenticate from a separate country that the user has not previously authenticated from.
+"""
+integration = ["okta"]
+uuid = "31585786-71f4-11ef-9e99-f661ea17fbcc"
+name = "Successful Impossible Travel Sign-On Events"
+language = ["ES|QL"]
+license = "Elastic License v2"
+notes = [
+    "`okta.actor.alternate_id` would be target of the threat adversary",
+    "Pivoting into a potential compromise requires an additional search for `okta.outcome.result` being `SUCCESS` for any `user.authentication*` value for `okta.event_type`",
+    "Pivot to any additional Okta logs after authentication to determine if activity is still being reported by separate countries."
+]
+mitre = ['T1078.004']
+query = ['''
+from logs-okta.system*
+| where @timestamp > NOW() - 7 day
+| where event.dataset == "okta.system"
+
+    // filter on successful sign-on events only
+    and okta.event_type == "policy.evaluate_sign_on"
+    and okta.outcome.result in ("ALLOW", "SUCCESS")
+
+// Truncate the timestamp to 15 minute intervals
```

Comment:
maybe bump it up to 1 hour or even 8hr, the likelihood of the adversary login in in the same 15m as the victim is a bit low.